rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -2155,3 +2155,32 @@ rocalNop(
     }
     return output;
 }
+
+RocalTensor ROCAL_API_CALL
+rocalPreEmphasisFilter(RocalContext p_context,
+                       RocalTensor p_input,
+                       bool is_output,
+                       RocalFloatParam p_preemph_coeff,
+                       RocalAudioBorderType preemph_border_type,
+                       RocalTensorOutputType output_datatype) {
+    Tensor* output = nullptr;
+    if ((p_context == nullptr) || (p_input == nullptr)) {
+        ERR("Invalid ROCAL context or invalid input tensor")
+        return output;
+    }
+    auto context = static_cast<Context*>(p_context);
+    auto input = static_cast<Tensor*>(p_input);
+    auto preemph_coeff = static_cast<FloatParam*>(p_preemph_coeff);
+    try {
+        RocalTensorDataType op_tensor_datatype = static_cast<RocalTensorDataType>(output_datatype);
+        TensorInfo output_info = input->info();
+        output_info.set_data_type(op_tensor_datatype);
```

Comment:
Can this augmentation support different data type outputs?
Do we have the condition check for this anywhere here or in OpenVX?

Reply:
No, all audio augmentations support only FP32 dtype except the Slice and Normalize kernels since they are generic and can be used for all the dtypes

Reply:
No, there is no specific condition in the OpenVX.

Reply:
TODO: THROW WRN

Reply:
DONE - Added a WRN here